modules/Bio/EnsEMBL/Analysis/Hive/Config/StarScallopRnaseq.pm

```diff
@@ -1041,15 +1261,74 @@ sub pipeline_analyses {
       -rc_name    => 'default',
     },
 
+    {
+      -logic_name => 'notification_pipeline_is_done_pcp',
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::MessagePipeline',
+      -parameters => {
+        messages   => [
+        {
+          url => $self->o('transcript_selection_url'),
+          logic_name => 'split_slices_on_intergenic',
+          param => 'input_gene_dbs',
+          data => $self->o('pcp_nr_db'),
+          update => 1,
+        },
+        {
+          url => $self->o('transcript_selection_url'),
+          logic_name => 'layer_annotation',
+          param => 'SOURCEDB_REFS',
+          data => $self->o('pcp_nr_db'),
+          update => 1,
+        },
+        {
+          url => $self->o('transcript_selection_url'),
+          logic_name => 'run_utr_addition',
+          param => 'donor_dbs',
+          data => $self->o('pcp_nr_db'),
+          update => 1,
+        },
+        {
+          url => $self->o('transcript_selection_url'),
+          logic_name => 'run_utr_addition_10GB',
+          param => 'donor_dbs',
+          data => $self->o('pcp_nr_db'),
+          update => 1,
+        },
+        {
+          url => $self->o('transcript_selection_url'),
+          logic_name => 'run_utr_addition_30GB',
+          param => 'donor_dbs',
+          data => $self->o('pcp_nr_db'),
+          update => 1,
+        },
+        {
+          url => $self->o('transcript_selection_url'),
+          logic_name => 'utr_memory_failover',
+          param => 'donor_dbs',
+          data => $self->o('pcp_nr_db'),
+          update => 1,
```

Comment:
I could be wrong but I don't think it is needed as we already add the "scallop" database for the donor array. I would expect the data to be the same